src/saving/FileHelpers.cs

```diff
@@ -43,4 +44,19 @@ public static bool Exists(string path)
         using var directory = new Directory();
         return directory.FileExists(path);
     }
+
+    public static Error TryCreateWrite(string name)
+    {
+        using var file = new File();
+        var e = file.Open(Constants.SAVE_FOLDER + "/" + name, File.ModeFlags.Write);
```

Comment:
Use `Path.Combine`

Reply:
What? No such func.

Reply:
It's used elsewhere for example: https://github.com/Revolutionary-Games/Thrive/blob/b11b68630cfdc84c1bd8bd812682bb47f0947e09/src/modding/ModLoader.cs#L65
I'm pretty sure nowhere else is this kind of not fully correct path combining done, elsewhere Path.Combine is used.

Reply:
Turns out we had the old PathUtils from when I didn't know that Path.Combine existed... I made a PR to remove that: https://github.com/Revolutionary-Games/Thrive/pull/3091